qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/XssOnPhoneNumberFieldSteps.java

```diff
@@ -0,0 +1,117 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.ActiveVisitsPage;
+import org.openmrs.contrib.qaframework.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.contrib.qaframework.page.RegistrationEditSectionPage;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class  XssOnPhoneNumberFieldSteps  extends Steps {
+    
+    private TestData.PatientInfo patient;
+    private ActiveVisitsPage activeVisitsPage;
+    private ClinicianFacingPatientDashboardPage patientDashboardPage;
+    private RegistrationEditSectionPage registrationEditSectionPage;
+
+    @Before(RunTest.HOOK.SELENIUM_XSS_VULNERABILITY)
+    public void setUp() throws Exception {
+        initiateWithLogin();
+        patient = createTestPatient();
+        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
+    }
+
+    @After(RunTest.HOOK.SELENIUM_XSS_VULNERABILITY)
+    public void tearDown() throws Exception {
+        deletePatient(patient);
```

Comment:
how is the test quitting  the run?

Reply:
As you can see the method used above. After all the functionalities are done., the test is supposed to delete that patient and quits the browser.

Reply:
could you point to the method that makes the test to quit the run?

Reply:
> could you refer to the method that makes the test to quit the run?

@kdaud  Its not by default that the test is supposed to quit using **quit()** method. the same exact method . delete() quits the browser.

Reply:
where is the delete() function that you are referring to?

Reply:
Resolved thanks.

Reply:
Responded to your question with a commit.